extensions/amp-imgur/0.1/validator-amp-imgur.protoascii

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-imgur
+  html_format: AMP
+  tag_name: "SCRIPT"
+  satisfies: "amp-imgur extension .js script"
+  requires: "amp-imgur"
+  extension_spec: {
+    name: "amp-imgur"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-imgur>
+  html_format: AMP
+  tag_name: "AMP-IMGUR"
+  satisfies: "amp-imgur"
+  requires: "amp-imgur extension .js script"
+  attrs: {
+    name: "data-imgurid"
```

Comment:
I noticed several of your examples use `data-imgur-id` and not `data-imgurid`, so lets update this to `data-imgur-id` here on line 35.